packages/component/src/Localization/ar-EG.js

```diff
@@ -0,0 +1,103 @@
+/* eslint no-magic-numbers: ["error", { "ignore": [1, 5, 24, 48, 60000, 3600000] }] */
+
+import getLocaleString from './getLocaleString';
+
+function xMinutesAgo(dateStr) {
+  const date = new Date(dateStr);
+  const dateTime = date.getTime();
+
+  if (isNaN(dateTime)) {
+    return dateStr;
+  }
+
+  const now = Date.now();
+  const deltaInMs = now - dateTime;
+  const deltaInMinutes = Math.floor(deltaInMs / 60000);
+  const deltaInHours = Math.floor(deltaInMs / 3600000);
+  const countofMinutesIsTen = 10; /* used for arabic translation only */
+
+  if (deltaInMinutes < 1) {
+    return 'حالا';
+  } else if (deltaInMinutes === 1) {
+    return 'منذ دقيقة';
+  } else if ((deltaInHours < 1) & (deltaInMinutes <= countofMinutesIsTen)) {
+    return `منذ ${deltaInMinutes} دقائق`;
+  } else if (deltaInHours < 1) {
+    return `منذ ${deltaInMinutes} دقيقة`;
+  } else if (deltaInHours === 1) {
+    return `منذ ساعة`;
+  } else if (deltaInHours < 5) {
+    return `منذ ${deltaInHours} ساعات`;
+  } else if (deltaInHours <= 24) {
+    return `اليوم`;
+  } else if (deltaInHours <= 48) {
+    return `أمس`;
+  }
+  return getLocaleString(date, 'ar-EG');
+}
+
+function botSaidSomething(avatarInitials, text) {
+  // return `Bot ${avatarInitials} said, ${text}`;
+  return `البوت ${avatarInitials} قال, ${text}`;
+}
+
+function downloadFileWithFileSize(downloadFileText, fileName, size) {
+  // Full text should read: "Download file <filename> of size <filesize>"
+  // return `${downloadFileText} ${fileName} of size ${size}`;
+  return `تحميل ملف ${fileName} بحجم ${size}`;
+}
+
+function uploadFileWithFileSize(fileName, size) {
```

Comment:
I just made a fix to this function today. Could you change it to the following? 


```
function uploadFileWithFileSize(uploadFileText, fileName, size) {
  return `${fileName} بحجم ${size}`; // Please add ${uploadFileText} where appropriate
}
```

Reply:
applied in commit https://github.com/microsoft/BotFramework-WebChat/pull/2593/commits/cb33bca6293887db0c9b117b8f5815ff487c28dc